airbyte-scheduler/app/src/main/java/io/airbyte/scheduler/app/JobScheduler.java

```diff
@@ -92,19 +91,20 @@ public void run() {
   }
 
   private void scheduleSyncJobs() throws IOException {
-    final AtomicInteger jobsScheduled = new AtomicInteger();
+    int jobsScheduled = 0;
     final List<StandardSync> activeConnections = getAllActiveConnections();
 
     for (StandardSync connection : activeConnections) {
       final Optional<Job> previousJobOptional = jobPersistence.getLastReplicationJob(connection.getConnectionId());
 
       if (scheduleJobPredicate.test(previousJobOptional, connection)) {
         jobFactory.create(connection.getConnectionId());
+        jobsScheduled++;
       }
     }
-    int jobsScheduledCount = jobsScheduled.get();
-    if (jobsScheduledCount > 0) {
-      LOGGER.info("Job-Scheduler Summary. Active connections: {}, Jobs scheduler: {}", activeConnections.size(), jobsScheduled.get());
+
+    if (jobsScheduled > 0) {
```

Comment:
@tuliren looks like you were the last person to touch this - do these changes preserve the intent? I'm somewhat guessing here :)

Reply:
I think so. The purpose of my change is to only creating this log if there are non zero jobs scheduled. This should still work.